src/reducers/issuesReducer/issuesReducer.js

```diff
@@ -0,0 +1,15 @@
+import types from '../../actions/actionTypes';
+import initialState from '../initialState';
+
+export default function (state = initialState.issues, action) {
+    switch (action.type) {
+
+    case types.UPDATE_ISSUES: {
+        const newState = JSON.parse(JSON.stringify(state));
```

Comment:
Since we do `JSON.parse(JSON.stringify(state))` so much. I wonder if we should make a utility for this. Might help as we will be doing that more and more.